detox/android/detox/build.gradle

```diff
@@ -62,11 +65,11 @@ dependencies {
     implementation "org.jetbrains.kotlin:$_kotlinStdlib:$_kotlinVersion"
 
     // noinspection GradleDynamicVersion
-    compileOnly 'com.facebook.react:react-native:+'
+    compileOnly "com.facebook.react:react-native:${_reactNativeVersion}"
     // noinspection GradleDynamicVersion
-    compileOnly 'com.squareup.okhttp3:okhttp:+'
+    compileOnly "com.squareup.okhttp3:okhttp:${_okhttpVersion}"
     // noinspection GradleDynamicVersion
-    compileOnly 'com.squareup.okio:okio:+'
+    compileOnly "com.squareup.okio:okio:${okioVersion}"
```

Comment:
I was trying these changes locally, and I guess this should be `_okioVersion `.

Reply:
d4709bd55774cbdbf419d1d7ba6d3d9451289761